docs/cheaha/getting_started.md

```diff
@@ -74,6 +78,34 @@ You are on the login node if:
 
 - terminal prompt looks like `[<blazerid>@login001 ~]$`
 
+The Login node can be accessed from the Cheaha landing page or through the home directory, while compute nodes can be accessed after a job has been created on the "My Interactive Sessions". You can see in the images below, how to identify if you’re within a login node or compute node. Image below is a compute node. Safe for heavy computation.
+
+![!compute node terminal prompt shows username@c0112](images/comp_node.png)
+
+Compared to the image below which is a login node.
+
+![!login node terminal prompt shows username@login004](images/login_node.png)
+
+##### Slurm and Slurm Jobs
+
+Slurm Workload Manager is a widely used open-source job scheduler that manages the queue of jobs submitted to the compute nodes. It ensures efficient use of the cluster by allocating resources to jobs, prioritizing tasks, and managing queues of pending jobs. Starting Slurm jobs can be done in two primary ways: using Open OnDemand (OOD) or through the terminal. For more details on how to use Slurm on cheaha, please see our [slurm docs](..//cheaha/slurm/introduction.md).
+
+##### What Should Run in Jobs?
+
+Ideally, only non-intensive tasks like editing files, compiling programs, or managing job submissions should be performed on the login node. Compute-intensive tasks, large data analyses, and simulations should be submitted as Slurm jobs to compute nodes. This approach ensures that the login node remains responsive and available for all users to manage their tasks and submissions. Submitting compute-intensive tasks as Slurm jobs to compute nodes helps to prevent overloading the login node, ensuring a smoother experience for all users of the cluster. As Cheaha stakeholders we should hold ourselves to best practices, and ensure its optimal performance and accessibility.
```

Comment:
remove compiling programs

Reply:
Remove 'As Cheaha stakeholders we should hold ourselves to best practices, and ensure its optimal performance and accessibility.'